conifer/backends/vivadohls/hls-template/build_prj.tcl

```diff
@@ -50,10 +50,10 @@ if {$opt(synth)} {
 }
 
 if {$opt(cosim)} {
-    cosim_design -trace_level all
+    cosim_design -trace_level al
```

Comment:
For me Vivado complains about this change. If I leave it at `-trace_level all` it works.

Reply:
Yes, this is a mistake! I just changed it